src/middleware/deserializeUser.middleware.ts

```diff
@@ -15,7 +16,11 @@ import { verifyJWT } from "../utils/jwt.util";
 const deserializeUser = async (req: Request, res: Response, next: NextFunction) => {
 	const accessToken = (req.headers.authorization || "").replace(/^Bearer\s/, "");
 
-	if (!accessToken) return next();
+	if (!accessToken) {
+		return res.status(StatusCodes.UNAUTHORIZED).json({
+			error: "User is not logged in",
+		});
+	}
 
 	const decoded = await verifyJWT<{ _id: string }>(accessToken, "ACCESS_TOKEN_PUBLIC_KEY");
```

Comment:
Hey Ayush, Could you make this change and turn `ACCESS_TOKEN_PUBLIC_KEY` into a environemnt variable. Thank You!

Reply:
It's already in ENV